src/DependencyInjection/Compiler/PageResolverBlocksTypesPass.php

```diff
@@ -20,7 +21,7 @@ public function process(ContainerBuilder $container)
             return;
         }
 
-        $pageBlocksIndexFile = EzSystemsEzPlatformGraphQLExtension::SCHEMA_DIR . '/PageBlocksList.types.yml';
+        $pageBlocksIndexFile = EzSystemsEzPlatformGraphQLBundle::EZPLATFORM_CONFIG_DIR . '/PageBlocksList.types.yml';
```

Comment:
Can we inject `$schemaDir` in the constructor same as `RegisterBlocksTypesPass` to streamline it? This way we can change it in the same place.

Reply:
I'll check, but yes, I think we can 👍